addons/hr/tests/test_hr_employee.py

```diff
@@ -161,6 +161,33 @@ def test_employee_member_of_department(self):
             employees.filtered_domain(['!'] + employees._search_part_of_department('in', [True])),
             emp_other + emp_parent)
 
+    def test_batch_user_creation_skips_employees_with_existing_users_email(self):
```

Comment:
Now that the other PR has been merged, would rename to ``test_user_creation_from_employee_multi_emails`` so that it can be placed with related ``test_user_creation_from_employee_with_invalid_email``